src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/PropertyStoreTests.cs

```diff
@@ -0,0 +1,204 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#nullable enable
+
+using System.Drawing;
+
+namespace System.Windows.Forms.Tests;
+
+public class PropertyStoreTests
+{
+    private static readonly int s_bool = PropertyStore.CreateKey();
+    private static readonly int s_byte = PropertyStore.CreateKey();
+    private static readonly int s_sbyte = PropertyStore.CreateKey();
+    private static readonly int s_char = PropertyStore.CreateKey();
+    private static readonly int s_decimal = PropertyStore.CreateKey();
+    private static readonly int s_double = PropertyStore.CreateKey();
+    private static readonly int s_float = PropertyStore.CreateKey();
+    private static readonly int s_int = PropertyStore.CreateKey();
+    private static readonly int s_uint = PropertyStore.CreateKey();
+    private static readonly int s_long = PropertyStore.CreateKey();
+    private static readonly int s_ulong = PropertyStore.CreateKey();
+    private static readonly int s_short = PropertyStore.CreateKey();
+    private static readonly int s_ushort = PropertyStore.CreateKey();
+    private static readonly int s_object = PropertyStore.CreateKey();
+    private static readonly int s_color = PropertyStore.CreateKey();
+    private static readonly int s_formWindowState = PropertyStore.CreateKey();
+
+    public static IEnumerable<object[]> PropertyStore_TryGetValue_Exists_TestData()
+    {
+        yield return new object[] { s_bool, true };
+        yield return new object[] { s_byte, (byte)1 };
+        yield return new object[] { s_sbyte, (sbyte)-1 };
+        yield return new object[] { s_char, 'a' };
+        yield return new object[] { s_decimal, 1.0m };
+        yield return new object[] { s_double, 1.0d };
+        yield return new object[] { s_float, 1.0f };
+        yield return new object[] { s_int, 1 };
+        yield return new object[] { s_uint, (uint)1 };
+        yield return new object[] { s_long, 1L };
+        yield return new object[] { s_ulong, 1UL };
+        yield return new object[] { s_short, (short)1 };
+        yield return new object[] { s_ushort, (ushort)1 };
+        yield return new object[] { s_object, new object() };
+        yield return new object[] { s_color, Color.Red };
+        yield return new object[] { s_formWindowState, FormWindowState.Maximized };
+    }
+
+    [WinFormsTheory]
+    [MemberData(nameof(PropertyStore_TryGetValue_Exists_TestData))]
+    public void PropertyStore_TryGetValue_Exists(int key, object? value)
+    {
+        PropertyStore properties = new PropertyStore();
```

Comment:
Consider the following change across the file: 
```suggestion
        PropertyStore store = new();
```